api/metrics_service.proto

```diff
@@ -5,6 +5,7 @@ syntax = "proto3";
 package envoy.api.v2;
 
 import "api/base.proto";
+import "api/grpc_cluster.proto";
 
 import "google/api/annotations.proto";
```

Comment:
Is this file needed?

Reply:
Good Catch. This is not required. I will remove this.